src/Storage/Storage.Management.Test/ScenarioTests/StorageDataPlaneTests.ps1

```diff
@@ -671,6 +671,152 @@ function Test-Common
     }    
 }
 
+<#
+    .SYNOPSIS
+    Tests DatalakeGen-only related commands.
+#>
+function Test-DatalakeGen2
+{
+    Param(
+        [Parameter(Mandatory = $True)]
+        [string]
+        $StorageAccountName,
+        [Parameter(Mandatory = $True)]
+        [string]
+        $ResourceGroupName
+    ) 
+
+    New-TestResourceGroupAndStorageAccount -ResourceGroupName $ResourceGroupName -StorageAccountName $StorageAccountName -EnableHNFS $true
```

Comment:
Seems these case doesn't contain the case that operate on the file or dir that without permision.

Reply:
Actually, for the operation  contains permission as input , like new/move of file and directory, each runs 2 times , one with permission, one without permission. And all files and directory’s has permission even user not set it.